addons/web_tour/static/src/debug/tour_dialog_component.js

```diff
@@ -41,3 +41,4 @@ export default class ToursDialog extends Component {
 ToursDialog.template = "web_tour.ToursDialog";
 ToursDialog.components = { Dialog };
 ToursDialog.title = _t("Tours");
+ToursDialog.confirmLabel = _t("Close");
```

Comment:
Same remark here. I'm not sure it's necessary to have a variable for the confirm label of the dialog if we don't reuse the component elsewhere.

Reply:
Same for this one :)